apps/foundation-demo/src/app/design/molecules/daff-footer/components/daff-footer-category/daff-footer-category.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
I think this component is unncessary. You could probably get away with a `<ng-content select="daff-footer-icons"></ng-content>` and completely remove this component.

Reply:
I disagree (also assuming you meant to comment on the daff-footer-icons file). If I remove this, then the styling of the icons needs to move into the foundation-demo implementation. That means that the styling of the icons from the developer needs to agree with the grid that is defined by daff-footer, OR I need to remove the grid stylings from daff-footer. But that makes the daff-footer component completely useless. So, I think daff-footer and its components need to be responsible for most (if not all) of the styling of the footer, or it should just go away.

Reply:
@lderrickable So, my thought here is that the `daff-footer` (much like the `daff-header`) defines some simple colors and paddings and potentially a basic grid. Ideally I don't want to be TOO opinionated by saying in a "daff-footer" `x` goes here, `y` goes here and `z` goes here. But I think there may be value to setting a basic color and some padding.

Reply:
@lderrickable we're going to remove this component as discussed later on in the thread.